CardDictionaries/Dynasty/DYNShared.php

```diff
@@ -599,6 +599,9 @@ function HasSurge($cardID)
   switch($cardID) {
     case "DYN194": case "DYN195": case "DYN197": case "DYN198": case "DYN199":
     case "DYN203": case "DYN204": case "DYN205": case "DYN206": case "DYN207": case "DYN208":
+    case "ROS207": case "ROS208": case "ROS209": case "ROS176": case "ROS177": case "ROS178":
+    case "ROS198": case "ROS199": case "ROS200": case "ROS189": case "ROS190": case "ROS191":
+    case "ROS201": case "ROS202": case "ROS203":
```

Comment:
same thing as before with ascending order of set number and I do think it just reads better if there are three cards per line with like card names.